### PR TITLE
ProjectionAndSingleSTDCM

### PR DESCRIPTION
**Pull request by alexandredamiron**

### CODE REVIEW

**Comment by alexandredamiron** [comment]
This will ensure that only one stdcm (last called is shown, that GET use its projection). It also increase the size of the GET in stdcm context

File: front/src/applications/stdcm/views/StdcmRequestModal.js [hunk 1]
```diff
@@ -6,21 +6,21 @@ import {
...
 import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
 import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
 import PropTypes from 'prop-types';
 import ReactModal from 'react-modal';
-import axios from 'axios';
 // OSRD helpers
 import createTrain from 'applications/osrd/components/Simulation/SpaceTimeChart/createTrain';
 import formatStdcmConf from 'applications/stdcm/formatStcmConf';
-import { post } from 'common/requests';
+import { post } from 'common/requests.ts';
```
Comment (anisometropie) [inline front/src/applications/stdcm/views/StdcmRequestModal.js]: .ts to remove

File: front/src/applications/stdcm/views/StdcmRequestModal.js [hunk 2]
```diff
@@ -100,6 +104,14 @@ export default function StdcmRequestModal(props) {
           dispatch(updateConsolidatedSimulation(consolidatedSimulation));
           dispatch(updateSimulation(newSimulation));
           dispatch(updateSelectedTrain(newSimulation.trains.length - 1));
+          /*
```
Comment (anisometropie) [inline front/src/applications/stdcm/views/StdcmRequestModal.js]: comment to remove

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/src/applications/stdcm/views/StdcmRequestModal.js (hunk 1 above)
Comment (alexandredamiron) [reply front/src/applications/stdcm/views/StdcmRequestModal.js]: done

File: front/src/applications/stdcm/views/StdcmRequestModal.js (hunk 2 above)
Comment (alexandredamiron) [reply front/src/applications/stdcm/views/StdcmRequestModal.js]: done

**Review by anisometropie — APPROVED** [review]

**Pull request merged**